src/openApi/apiRequestModel.ts

```diff
@@ -1,5 +1,5 @@
-export interface ApiRequest<T> {
-  body?: T;
+export interface ApiRequest<Body = any, Query = any> {
```

Comment:
Soo initially the `T` was here to type the body, but then we needed to type the query params so (problably me) used T for either body or query. And this won't work for endpoints with both query & body and also having just `T` wasn't very clear that it's either body or query.

Reply:
Yeah :)